client/src/resources/lang/en_us.json

```diff
@@ -758,9 +758,18 @@
     "phase.night:var.0": "Nighttime",
     "phase.night.subtitle": "Use your abilities to help your team.",
 
+    "phase.recess": "Recess",
+    "phase.recess:var.0": "Intermission",
+    "phase.recess:var.1": "Aftermath",

```

Comment:
I wouldn't do this. Each phase should probably have exactly 1 name. If you add pausing i guess it makes sense. a bit more, but I don't like. I don't feel strongly about this.

Reply:
Fair, I'll do this